### PR TITLE
Fix test client agg multi range

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -8,10 +8,18 @@ import (
 )
 
 var client = NewClient("localhost:6379", "test_client", MakeStringPtr("SUPERSECRET"))
+var dummy = client.FlashAll()
 
 var defaultDuration, _ = time.ParseDuration("1h")
 var tooShortDuration, _ = time.ParseDuration("10ms")
 
+func (client *Client) FlashAll() (err error) {
```
Comment (RoeyPrat) [inline client_test.go]: flash or flush?

File: client_test.go [hunk 2]
```diff
@@ -8,10 +8,18 @@ import (
 )
 
 var client = NewClient("localhost:6379", "test_client", MakeStringPtr("SUPERSECRET"))
+var dummy = client.FlashAll()
```
Comment (RoeyPrat) [inline client_test.go]: do we need this dummy? why not use a [blank identifier](https://golang.org/doc/effective_go.html#blank)?

Comment (ashtul) [reply client_test.go]: because this is the first time I'm writing in golang :)

**Comment by RoeyPrat** [comment]
commit messages could be a bit more descriptive...
something like "in version X of redistimeseries we moved from seconds to milliseconds, updating this change in unit test Y"
"running flushall before unit tests"

File: client_test.go (hunk 2 above)
Comment (RoeyPrat) [reply client_test.go]: cool 👍

File: client_test.go [hunk 3]
```diff
@@ -8,10 +8,18 @@ import (
...
 
 var defaultDuration, _ = time.ParseDuration("1h")
 var tooShortDuration, _ = time.ParseDuration("10ms")
 
+func (client *Client) FlashAll() (err error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	_, err = conn.Do("FLUSHALL")
```
Comment (RoeyPrat) [inline client_test.go]: If FLUSHALL is not supported by redigo, I would suggest opening a PR there :)

Comment (ashtul) [reply client_test.go]: @gkorland @RoeyPrat 
Not sure why exactly the coverage is down but we should change the limitation on this or make a one-time exception

Comment (RoeyPrat) [reply client_test.go]: It should not block you from merging. 
If circle ci passes and a maintainer gives you a +1, you should be able to merge.

Comment (ashtul) [reply client_test.go]: > If circle ci passes and a maintainer gives you a +1, you should be able to merge.
@RoeyPrat  Would you do me the honor?

Comment (RoeyPrat) [reply client_test.go]: In my opinion, the commit messages are uninformative. see comment https://github.com/RedisTimeSeries/redistimeseries-go/pull/29#issuecomment-524808098

Comment (ashtul) [reply client_test.go]: @RoeyPrat Done!

Comment (ashtul) [reply client_test.go]: > In my opinion, the commit messages are uninformative. see comment [#29 (comment)](https://github.com/RedisTimeSeries/redistimeseries-go/pull/29#issuecomment-524808098)

@RoeyPrat  We are going to squash this anyway and have one comment for it all.

Comment (RoeyPrat) [reply client_test.go]: image
these are the commits I'm seeing right now. 
1. is RTS the acronym of redis timeseries module? 
2. "minor changes" is, in my opinion, not an informative commit message.

**Comment by ashtul** [comment]
**Bug fixes**
- `key` changed to `key2` which crashed the test
- Change of seconds resolution to milliseconds resolution to reflect RedisTimeSeries changes.

**Enhancement**
- Add `flushall` command before tests to allow consecutive tests 

@RoeyPrat

**Pull request merged**